packages/thyra/core/src/interfaces/nonlinear/model_evaluator/fundamental/Thyra_ModelEvaluatorBase_def.hpp

```diff
@@ -94,7 +94,7 @@ condCloneVec_mp(
 
 template<class Scalar>
 ModelEvaluatorBase::InArgs<Scalar>::InArgs()
-  :modelEvalDescription_("WARNING!  THIS INARGS OBJECT IS UNINITALIZED!")
+  :modelEvalDescription_("WARNING!  THIS INARGS OBJECT IS UNINITIALIZED!")
```

Comment:
This technically changes behavior.  If there were good unit tests for this software, it would break those unit tests.  But this class was written years before I read "Working Effectively with Legacy Code" or "Unit Testing" so it likely will not break any automated tests (both a good thing and a bad thing at the same time).

Reply:
"Accessing an uninitialized object is undefined behavior" -- there, you're done :D

Reply:
> "Accessing an uninitialized object is undefined behavior" -- there, you're done :D

That state does not mean that the memory is uninitialized, just that the object has not been given a useful state.  But since only a unit test would trigger this state (and the automated tests passed), then we can assume that no unit test checks this (which is not surprising given when this code was written).

I will go ahead and click the merge button.